### PR TITLE
Adding attributes for v1 endpoints

### PR DESCRIPTION
**Pull request by komalPandey152**

### CODE REVIEW

**Comment by JarbasHorst** [comment]
Hello @komalPandey152, I reviewed your PR and left some comments. Please, take a look when you will have time.

File: api-reference/v1.0/resources/bookingcustomerinformation.md [hunk 1]
```diff
@@ -18,14 +18,15 @@ Inherits from [bookingCustomerInformationBase](bookingcustomerinformationbase.md
...
+|customQuestionAnswers|[bookingQuestionAnswer](../resources/bookingquestionanswer.md) collection|It consists of the list of custom questions and answers given by the customer as part of the appointment |
+|emailAddress|String| The SMTP address of the [bookingCustomer](../resources/bookingcustomer.md) who is booking the appointment |
 |location|[location](../resources/location.md)| Represents location information for the bookingCustomer who is booking the appointment. |
 |name|String|The customer's name. |
 |notes|String|Notes from the customer associated with this appointment. You can get the value only when reading this **bookingAppointment** by its ID. You can set this property only when initially creating an appointment with a new customer. After that point, the value is computed from the customer represented by the **customerId**. |
 |phone|String|The customer's phone number. |
+|smsNotificationsEnabled|Boolean|Indicates whether the customer will receive SMS notifications for the appointment.|
```
Comment (JarbasHorst) [inline api-reference/v1.0/resources/bookingcustomerinformation.md:28]: In the CSDL file, I can't find the property **smsNotificationsEnabled** as part of the bookingCustomerInformation resource. I can see however that this property is part of the BETA bookingCustomerInformation. Please, ensure that the property smsNotificationsEnabled needs to be added to the v1.0 bookingCustomerInformation resource topic.

File: api-reference/v1.0/resources/bookingcustomer.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Inherits from [bookingCustomerBase](bookingcustomerbase.md).
 |[List customers](../api/bookingbusiness-list-customers.md) | [bookingCustomer](bookingcustomer.md) collection | Get a list of **bookingCustomer** objects. |
 |[Create bookingCustomer](../api/bookingbusiness-post-customers.md) | [bookingCustomer](bookingcustomer.md) | Create a new **bookingCustomer** object. |
 |[Get bookingCustomer](../api/bookingcustomer-get.md) | [bookingCustomer](bookingcustomer.md) |Read the properties and relationships of a **bookingCustomer** object.|
-|[Update](../api/bookingcustomer-update.md) | [bookingCustomer](bookingcustomer.md)	|Update a **bookingCustomer** object. |
+|[Update](../api/bookingcustomer-update.md) | None	|Update a **bookingCustomer** object. |
```
[the remark this answers is not in the document]
Comment (komalPandey152) [reply api-reference/v1.0/resources/bookingcustomer.md:25]: I have verified. Update call returns 200 OK status code. Reverting this change.

File: api-reference/v1.0/resources/bookingcustomquestion.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Represents a custom question for a [bookingBusiness](bookingbusiness.md).
...
 | [Create bookingCustomQuestion](../api/bookingbusiness-post-customquestions.md) | [bookingCustomQuestion](../resources/bookingcustomquestion.md)            | Create a new [bookingCustomQuestion](../resources/bookingcustomquestion.md) object.                               |
 | [Get bookingCustomQuestion](../api/bookingcustomquestion-get.md)               | [bookingCustomQuestion](../resources/bookingcustomquestion.md)            | Read the properties and relationships of a [bookingCustomQuestion](../resources/bookingcustomquestion.md) object. |
-| [Update bookingCustomQuestion](../api/bookingcustomquestion-update.md)         | [bookingCustomQuestion](../resources/bookingcustomquestion.md)            | Update the properties of a [bookingCustomQuestion](../resources/bookingcustomquestion.md) object.                 |
+| [Update bookingCustomQuestion](../api/bookingcustomquestion-update.md)         | None | Update the properties of a [bookingCustomQuestion](../resources/bookingcustomquestion.md) object.                 |
```
[the remark this answers is not in the document]
Comment (komalPandey152) [reply api-reference/v1.0/resources/bookingcustomquestion.md:23]: I have verified. Update call returns 200 OK status code. Reverting this change.

File: api-reference/v1.0/resources/bookingcustomerinformation.md (hunk 1 above)
Comment (komalPandey152) [reply api-reference/v1.0/resources/bookingcustomerinformation.md:30]: Yes, this was made by mistake. Reverting it.

File: api-reference/v1.0/api/bookingbusiness-list-appointments.md [hunk 4]
```diff
@@ -130,6 +130,7 @@ Content-type: application/json
             "price": 10,
             "serviceNotes": null,
             "optOutOfCustomerEmail": false,
+            "anonymousJoinWebUrl": null,
```
[the remark this answers is not in the document]
Comment (komalPandey152) [reply api-reference/v1.0/api/bookingbusiness-list-appointments.md:133]: done

File: api-reference/v1.0/resources/bookingservice.md [hunk 5]
```diff
@@ -36,8 +36,10 @@ Represents information about a particular service provided by a [bookingBusiness
 |description|String|A text description for the service.|
 |displayName|String|A service name.|
 |id|String|The ID of that service, in a GUID format. Read-only.|
```
[the remark this answers is not in the document]
Comment (komalPandey152) [reply api-reference/v1.0/resources/bookingservice.md:38]: webrtcUrl is the anonymousJoinWebUrl

File: api-reference/v1.0/api/bookingbusiness-post-services.md [hunk 6]
```diff
@@ -132,7 +134,7 @@ Content-type: application/json
     "id": "57da6774-a087-4d69-b0e6-6fb82c339976",
```
[the remark this answers is not in the document]
Comment (komalPandey152) [reply api-reference/v1.0/api/bookingbusiness-post-services.md:134]: done

File: api-reference/v1.0/api/bookingservice-update.md [hunk 7]
```diff
@@ -53,6 +53,7 @@ PATCH /solutions/bookingBusinesses/{id}/services/{id}
 |description|String|A text description for the service.|
 |displayName|String|A service name.|
 |id|String| The unique identifier for the **bookingService**. Read-only.|
+|languageTag|String|The language of the self-service booking page.|
 |isHiddenFromCustomers|Boolean|If `true`, the service is not available to customers for booking.|
```
[the remark this answers is not in the document]
Comment (komalPandey152) [reply api-reference/v1.0/api/bookingservice-update.md:57]: added

**Review by Lauragra — APPROVED** [review]

**Pull request merged**